Desktop_Interface/isobufferbuffer.cpp

```diff
@@ -1,49 +1,114 @@
 #include "isobufferbuffer.h"
 
-isoBufferBuffer::isoBufferBuffer(uint32_t length)
-    : bufferLength(length)
-{
-    mid = bufferLength/2;
-    buffer = (char *) malloc((bufferLength * 3) / 2);
+/* isoBufferBuffer is implemented as two consecutive, duplicate,
```

Comment:
Good idea leaving a comment like this. Makes understanding everything much easier when you haven't looked at it in 2 years. Definitely should have written something like this at the time.